documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -132,3 +132,44 @@ broken special characters.
 
 image::constraints-decision-tree.svg[width=100%]
 
+[[reuse-elements]]
+== Reuse Elements
+
+To ensure efficiency and maintainability, it is essential to reuse these elements across different Aspects
+*_WHENEVER_* possible.
+
+`xref:ROOT:entities.adoc#entities[Entities]`, `xref:ROOT:characteristics.adoc#characteristics[Characteristics]`,
+`xref:ROOT:modeling-guidelines.adoc#declaring-events[Events]`, `xref:ROOT:modeling-guidelines.adoc#declaring-constraints[Constraints]`
+and `xref:ROOT:modeling-guidelines.adoc#declaring-properties[Properties]` are fundamental components of an Aspect.
+Reusing these elements across different Aspects ensures consistency, reduces redundancy, and simplifies the management and
+integration of shared attributes and interactions.
+
+[[strict-ttl]]
```

Comment:
I'm not sure we should keep this section at all. Firstly, "Strict TTL" is a working title for a set of clarifications. Secondly, the clarifications have already been added in form of an ADR in https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/pull/298, since it is mainly relevant for implementors of SAMM-based tooling, not for modellers.
_If_ we want to make this list an "official" part of the specification, all the definitions (such as, e.g., "aspect model file") given in the ADR must also first be made a part of the spec. For this, the appendix or best practices guide would not the right place.
@chris-volk What do you think?

Reply:
This section can be removed. Yes, strict ttl is just a working title. 
Consolidation of ttl usage across tools (SAMM CLI, Aspect Model Editor) shall happen after implementation.